tests/demo-todo-app.spec.ts

```diff
@@ -339,4 +343,5 @@ async function createDefaultTodos(page: Page) {
     await page.locator(".new-todo").fill(item);
     await page.locator(".new-todo").press("Enter");
   }
+  await expect(page.locator(".todo-list li")).toHaveCount(3);
```

Comment:
Making sure all 3 items are added to the todo list reduces/eliminates test flake where tests rely on all 3 items being present before proceeding to the next step.